lametro/views.py

```diff
@@ -585,11 +590,32 @@ def dispatch(self, request, *args, **kwargs):
 
         return response
 
+    def post(self, request, *args, **kwargs):
+        slug = self.kwargs['slug']
+        person = self.model.objects.get(slug=slug)
+        self.object = self.get_object()
```

Comment:
What's going on here?

Reply:
So for this, I was getting an error along the lines of `LAPersonDetailView object has no attribute 'object'` and after looking it up found that people were having success with adding `self.object = self.get_object()` to assign an object for `get_context_data()` to use. 

I later found that on line 119 someone had the same issue and [linked to this SO answer](https://stackoverflow.com/questions/34460708/checkoutview-object-has-no-attribute-object). So maybe this is just a quirk with django?